R/MakeKRICharts.R

```diff
@@ -5,46 +5,46 @@
 #' @param strWorkflowId `character` Name of workflow ID, e.g., 'kri0001'.
 #' @param lData `list` A list containing two data frames - dfSummary and dfBounds.
 #' @param lStackedSnapshots `list` A list returned by [gsm::Make_Snapshot()] containing flat files for snapshot/assessment data.
-#' @param dfMetaWorkflow `data.frame` Workflow metadata. See [gsm::meta_workflow].
+#' @param dfWorkflow `data.frame` Workflow metadata. See [gsm::meta_workflow].
 #'
 #' @return A list (lCharts) containing three charts - scatterJS, barMetricJS, and barScoreJS.
 #'
 #'
 #' @export
-MakeKRICharts <- function(strWorkflowId = NULL, lData = NULL, lStackedSnapshots = NULL, dfMetaWorkflow = NULL) {
+MakeKRICharts <- function(strWorkflowId = NULL, lData = NULL, dfWorkflow = NULL, lStackedSnapshots = NULL) {
```

Comment:
```suggestion
MakeKRICharts <- function(strWorkflowId = NULL, lData = NULL, lLabels = NULL, lStackedSnapshots = NULL) {
```

This is really the key change I'm proposing. I think it makes more sense to pass a list of labels instead of a data.frame with one row.